go.tera

```diff
@@ -0,0 +1,28 @@
+---
+whiskers:
+  version: "2.0.0"
+  matrix:
+    - flavor
+  filename: "{{flavor.identifier}}.go"
+---
+package catppuccingo
+
+// {{flavor.name}} variant
+type {{flavor.identifier}} struct{}
+
+// {{flavor.name}} flavor variant
+var {{flavor.identifier | title}} Flavor = {{flavor.identifier}}{}
+
+// {{flavor.name}}
+func ({{flavor.identifier}}) Name() string { return "{{flavor.identifier}}" }
+
+{% for _, color in flavor.colors %}
+// {{color.name}}
```

Comment:
Bit of a nitpick but I think it'd be nicer to remove this comment since the name of the function provides the same information. (Also makes the PR diff smaller :D)

Unless it's been added to act as the function docs?

```suggestion
```

Reply:
Disregard, read updated description